### PR TITLE
DM-27169: initial config files

### PR DESCRIPTION
**Pull request by r-owen**

### CODE REVIEW

File: COPYRIGHT [hunk 1]
```diff
@@ -0,0 +1 @@
+Copyright 2020 University of Washington
```
Comment (tribeiro) [inline COPYRIGHT:1]: Is this COPYRIGHT correct? Shouldn't it be something like:

```
Copyright 2020 Association of Universities for Research in Astronomy, Inc. (AURA)
```

?

File: tests/test_utils.py [hunk 2]
```diff
@@ -0,0 +1,63 @@
...
+    config_files = [path.name for path in config_dir.glob("*.xml")]
+    desired_config_files = [path.name for path in desired_config_dir.glob("*.xml")]
+    assert set(config_files) == set(desired_config_files)
+
+
+def test_get_qos_path() -> None:
+    qos_path = ddsconfig.get_qos_path()
+    assert qos_path.exists()
+    assert qos_path.is_file()
+    desired_qos_path = get_desired_pkg_root() / "qos" / "Qos.xml"
```
Comment (tribeiro) [inline tests/test_utils.py]: This should be `QoS.xml` no?

File: COPYRIGHT (hunk 1 above)
Comment (r-owen) [reply COPYRIGHT:1]: This is the correct copyright for repos that I make.

File: tests/test_utils.py (hunk 2 above)
Comment (r-owen) [reply tests/test_utils.py]: Yes. Thanks. Good catch.

**Pull request merged**